src/controllers/user.ts

```diff
@@ -0,0 +1,91 @@
+import { Request, Response, NextFunction } from 'express';
+import uuid from 'uuid/v1';
+import sequelize from '../data-access/sequelize';
+
+export const updateUserController = (req: Request, res: Response) => {
+    const { id, login, password, age } = req.body;
+
+    sequelize.models.user.findOne({
```

Comment:
Business logic should be moved to Services in order to be decoupled from controller

Reply:
Refactored, thank you